substrate/primitives/staking/src/offence.rs

```diff
@@ -37,29 +37,6 @@ pub type Kind = [u8; 16];
 /// so that we can slash it accordingly.
 pub type OffenceCount = u32;
 
-/// In case of an offence, which conditions get an offending validator disabled.
```

Comment:
I'm not sure this should go away, could you explain?

Reply:
Yes. `enum DisableStrategy` was used to determine if an offence should lead to a disablement or not. We don't need this anymore because each offence leads to disablement. Furthermore it causes confusion. Disabling strategy should mean how we handle offenders not if an offence leads to disabling or not. I agree that the last sentence is subjective but let's keep ambiguity to a minimum.

Reply:
Sounds good, thanks!